platform/windows/display_server_windows.cpp

```diff
@@ -2736,6 +2736,55 @@ void DisplayServerWindows::window_request_attention(WindowID p_window) {
 	FlashWindowEx(&info);
 }
 
+void DisplayServerWindows::window_set_taskbar_progress_value(float p_value, WindowID p_window) {
+	_THREAD_SAFE_METHOD_
+
+	ERR_FAIL_COND(!windows.has(p_window));
+	const WindowData &wd = windows[p_window];
+
+	if (pTaskbar == nullptr) {
+		CoCreateInstance(CLSID_TaskbarList, 0, CLSCTX_INPROC_SERVER, IID_ITaskbarList, (void **)&pTaskbar);
```

Comment:
Return of the `CoCreateInstance` should also be checked, it can fail:

```suggestion
		if (CoCreateInstance(CLSID_TaskbarList, 0, CLSCTX_INPROC_SERVER, IID_ITaskbarList, (void **)&pTaskbar) != S_OK) {
			pTaskbar = nullptr;
			return;
		}
```